`Development`: Add documentation for user experience (#5368)
ls1intum/Artemis: docs/user/user-experience/themes.rst

=====================
User Interface Themes
=====================

.. contents:: Content of this document
    :local:
    :depth: 2

Artemis ships both a light and a dark theme to improve your overall experience if you're used to dark themes, e.g. from
your preferred IDE.

Changing the theme
------------------

You can change the theme manually at any time by clicking the theme switcher icon in the navbar:

.. figure:: ./theme_switch.gif
    :alt: Click the icon in the navbar to change the theme

Synchronize the theme with your operating system
------------------------------------------------

> By default, Artemis will choose the dark theme if you indicated that you prefer dark themes in your OS settings.

Reviewer: ```suggestion
By default, Artemis will choose the preferred theme according to your OS settings.
```

Author: 244f84
